features/kuzzle.feature

```diff
@@ -564,6 +564,17 @@ Feature: Kuzzle functional tests
     Given I delete the profile with id "my-new-profile"
     Then I'm not able to find the profile with id "my-new-profile"
 
+  @security
+  Scenario: can't delete profile assigned to a user
+    When I create a user "useradmin" with id "useradmin-id"
+    Then I log in as useradmin:testpwd expiring in 1h
+    Then I create a new role "role1" with id "role1"
+    Then I create a new profile "profile1" with id "profile1"
+    Then I create a user "user1" with id "user1-id"
+    Then I'm not able to delete profile with id "profile1"
+    Then I'm able to find the profile with id "profile1"
+    Then I log out
```

Comment:
Is this necessary?

Reply:
Since I have to login at the beginning of the scenario (in order to create a role/profile/user) I found it just more coherent to end it by logging out

Reply:
The logout step is handled by Cucumber hooks I think. Usually, we do all the cleaning stuff in hooks, can you check if the tests still pass without the logout action?